src/main/AndroidManifest.xml

```diff
@@ -20,82 +20,85 @@ You should have received a copy of the GNU General Public License along with Sim
 @copyright 2015 Vojtech Kral
 -->
 <!-- package name and version codes are overwritten by Gradle -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    package="nl.mpcjanssen.simpletask" >
-
+    package="nl.mpcjanssen.simpletask"
+    >
     <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT" />
     <uses-permission android:name="android.permission.READ_CALENDAR" />
     <uses-permission android:name="android.permission.WRITE_CALENDAR" />
-
     <supports-screens
         android:anyDensity="true"
         android:largeScreens="true"
         android:normalScreens="true"
         android:resizeable="true"
         android:smallScreens="true"
-        android:xlargeScreens="true" />
-
+        android:xlargeScreens="true"
+        />
     <application
-            android:name=".TodoApplication"
-            android:allowBackup="true"
-            android:fullBackupContent="true"
-            android:icon="@drawable/ic_launcher"
-            android:label="@string/app_label"
-            android:theme="@style/AppTheme.Light.DarkActionBar" tools:ignore="UnusedAttribute">
-
+        android:name=".TodoApplication"
+        android:allowBackup="true"
+        android:fullBackupContent="true"
+        android:icon="@drawable/ic_launcher"
+        android:label="@string/app_label"
+        android:theme="@style/launchScreen"
+        >
+        <!-- tools:ignore="UnusedAttribute" -->
```

Comment:
Change the theme to `launchScreen`. The Simpletask activity sets the theme on launch anyway so launch is the only place affected by this change.

Commented out UnusedAttribute because doing so didn't seem to produce any errors or warnings. Didn't test it in studio though so it might be that...